lib/Image/Color.cpp

```diff
@@ -53,19 +54,19 @@ static double convertXYZ(double Val) {
   return Val > E ? pow(Val, 1.0 / 3.0) : (K * Val + 16.0) / 116.0;
 }
 
-static Color XYZToLAB(const Color Old) {
-  double X = convertXYZ(Old.R / D65WhitePoint.R);
-  double Y = convertXYZ(Old.G / D65WhitePoint.G);
-  double Z = convertXYZ(Old.B / D65WhitePoint.B);
+static Color xyzToLab(const Color Old) {
+  const double X = convertXYZ(Old.R / D65WhitePoint.R);
+  const double Y = convertXYZ(Old.G / D65WhitePoint.G);
+  const double Z = convertXYZ(Old.B / D65WhitePoint.B);
 
-  double L = fmax(0.0, 116.0 * Y - 16.0);
-  double A = 500 * (X - Y);
-  double B = 200 * (Y - Z);
+  const double L = fmax(0.0, 116.0 * Y - 16.0);
+  const double A = 500 * (X - Y);
+  const double B = 200 * (Y - Z);
   return Color(L, A, B, ColorSpace::LAB);
 }
 
 static double convertLAB(double Val) {
```

Comment:
Why didn't this get changed to `convertLab`?

Reply:
The camelCase rules don't perform dictionary lookups of words, so it doesn't know that "lab" is a word, which is great because in this case it is actually the L*a*b color space 😝 .